CRVResponse/src/MakeCrvRecoPulses.cc

```diff
@@ -68,18 +68,25 @@ void MakeCrvRecoPulses::RangeFinder(const std::vector<unsigned int> &waveform, c
   assert(peakStart>0);
   assert(peakEnd<waveform.size()-1);
 
+  //select a range of up to 4 points before and after the peak
+  //-find up to 5 points before and after the peak for which the waveform is stricly decreasing
+  //-remove 1 point on each side. this removes potentially "bad points" belonging to a second pulse (i.e. in double pulses)
   end=peakEnd+1;
-  for(size_t i=peakEnd; i<waveform.size()-1 && i<peakEnd+4; ++i)
+  start=peakStart-1;
```

Comment:
how do you know peakStart isn't 0?

Reply:
peakStart gets determined at another location where 0 is explicitly excluded.